blank/webpack.config.js

```diff
@@ -49,36 +50,28 @@ function sourceMap(mode) {
 
 const resolveConfig = {
   extensions: ['*', '.js', '.jsx', '.ts', '.tsx'],
-  alias: {
-    react: path.resolve(__dirname, 'node_modules', 'react'),
-    'styled-components': path.resolve(
-      __dirname,
-      'node_modules',
-      'styled-components'
-    ),
-  },
 }
 
 const babelLoaderConfig = {
-  test: /\.(js|jsx)$/,
-  exclude: /node_modules[\/\\](?!(@botonic)[\/\\])/,
+  test: /\.?(j|t)s?x/,
```

Comment:
why the "?" ?
shouldn't the second one go after the x?

Reply:
sure, good catch!